### ISSUE TITLE
txt?

### ISSUE THREAD

**Issue by CPScript** [Blank-c/Blank-Grabber#183]
it doesn't show the txt file under the discord message??? 

i dont know what this is happening, but it would be nice to get looked into :)

**Comment by CPScript** [comment]
i have used this once in the past, it gave me a zip file, and in the zip where txt files holding the users (my) info... now when i test this, it will not give me the zip???

**Comment by Blank-c** [comment]
Have you tried more than once?
Please contact me on discord and show me a screenshot.
Also, I'd recommend you to build it without Anti VM option and use it on a VM.

**Comment by CPScript** [comment]
> Have you tried more than once? Please contact me on discord and show me a screenshot. Also, I'd recommend you to build it without Anti VM option and use it on a VM.

Yes i have rebuilt it more than once... ill try to send you a screen shot as soon as i can
